docs/advanced-features/self-signed-certificates.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+To use a self-signed certificate with the `quilt3` API or CLI,
+the certificate must first be added to your Python environment.
+
+## Mac and Linux
+
+1. Determine the location of your certificates:
+    ```python
+    >>> import certifi
+    >>> certifi.where()
+    '/path/to/site-packages/certifi/cacert.pem'
+    ```
+2. Copy your custom self-signed certificate and paste into the
+`cacert.pem` file.
+
+## Windows
+
+1. Press «Win + R» to open a new command prompt and type `certmgr`.
+This opens the Windows Certificate Manager for the current user.
+2. Search all certificates stores for a certificate issued by `mitm`.
+There should be at least one certificate.
```

Comment:
The user will need to search for whatever certificate that's actually being used - I was just using "mitmproxy" in my test.

You can find the certificate issuer by opening any https website and looking at SSL settings:

![VirtualBox_Windows 8_21_04_2023_12_22_00](https://user-images.githubusercontent.com/4365877/233719613-597820cc-9c1a-4f7e-a9ef-a7f3afba7002.png)

Or more details here:

![VirtualBox_Windows 8_21_04_2023_12_22_34](https://user-images.githubusercontent.com/4365877/233719658-f5a2edb9-0e89-405b-a3ca-30c40f07423f.png)

(And of course this depends on the browser, etc.)